dev/dev-env-setup.sh

```diff
@@ -171,8 +171,8 @@ version_gt() {
 
 # Check if pyenv is installed and offer to install it if not present
 check_and_install_pyenv() {
-  pyenv_exist=$(command -v pyenv)
-
+  # `|| true` is for preventing scrip termination when `set -o errexit` bash flag is set.
+  pyenv_exist=$(command -v pyenv || true)
```

Comment:
Update.

Reply:
@WeichenXu123 could you also test if this would throw too and make the change in this PR if it's also broken? https://github.com/mlflow/mlflow/blob/10655f062b9431fa6533d7307411f3551a8827a1/dev/dev-env-setup.sh#L138

Reply:
same issue. updated